java-checks/src/main/java/org/sonar/java/checks/RedundantThrowsDeclarationCheck.java

```diff
@@ -37,32 +56,166 @@
 @RspecKey("S1130")
 public class RedundantThrowsDeclarationCheck extends IssuableSubscriptionVisitor {
 
+  public static final Set<String> SERIALIZABLE_CONTRACT_METHODS = ImmutableSet.of(
```

Comment:
Why public ? this is also a duplicate of  : ` org.sonar.java.checks.serialization.SerializableContract.SERIALIZABLE_CONTRACT_METHODS` is it intentional ?